src/components/anchor/code/templates/anchorTemplate.jsx

```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+
+module.exports = function() {
+
+  var dataAttributes = this.getDataAttributesFromProps();
+  var target = '_' + this.props.target;
+
+  return (
+    <a className="component-anchor" {...dataAttributes} title={this.props.title} href={this.props.href} onClick={this.props.handleClick} target={target}>
```

Comment:
Assuming that target is optional... this could break here as we could end up with `target=_`